FrameworkSystem/DB/InstalledComponentsDB.py

```diff
@@ -244,7 +244,7 @@ class HostLogging( componentsBase ):
   hostName = Column( 'HostName', String( 32 ), nullable = False, primary_key = True )
   # status
   DIRAC = Column( 'DIRACVersion', String( 32 ) )
-  Extension = Column( 'Extension', String( 64 ) )
+  Extensions = Column( 'Extensions', String( 64 ) )
```

Comment:
Well. I have updated the documentation: https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v6r15#framework-changes. It does not require to change it. If you do not change, you will not see the Extension installed in a certain host using SystemAdministrator.